tiri/src/lib.rs

```diff
@@ -34,10 +46,11 @@ use yktrace::tir::{
 const DUMP_CONTEXT: usize = 4;
 
 /// Each local variable is represented by one `Value` struct.
-/// This corresponds with the Miri's `Operand` type.
 #[derive(Debug, Clone, Eq, PartialEq)]
 enum Value {
+    /// A stand-alone immediate value.
     Immediate(Immediate),
+    /// Memory.
```

Comment:
Heap memory? Stack memory? Both?

Reply:
I believe both. Let's roll with both for now.